waku/v2/protocol/filter/subscribers_map.go

```diff
@@ -106,12 +95,12 @@ func (sub *SubscribersMap) Delete(peerID peer.ID, pubsubTopic string, contentTop
 
 	pubsubTopicMap, ok := sub.items[peerID]
 	if !ok {
-		return ErrNotFound
+		return subscription.ErrNotFound
```

Comment:
I wonder if we should instead create non-exported errors in this file, instead of reusing those from subscription package. Seems somewhat strange to have subscriber functions return subscription errors, but maybe it's too much bikeshedding :)